src/main/java/org/mozilla/jss/util/NativeErrcodes.java

```diff
@@ -246,12 +246,13 @@
     public static final int SEC_ERROR_DECRYPTION_DISALLOWED = 228;
     /**
      * @deprecated As of NSS 3.11, FORTEZZA is no longer supported.
-     * XP_SEC_FORTEZZA_BAD_CARD, XP_SEC_FORTEZZA_NO_CARD, 
+     * XP_SEC_FORTEZZA_BAD_CARD, XP_SEC_FORTEZZA_NO_CARD,
      * XP_SEC_FORTEZZA_NONE_SELECTED, XP_SEC_FORTEZZA_MORE_INFO
      * XP_SEC_FORTEZZA_PERSON_NOT_FOUND, XP_SEC_FORTEZZA_NO_MORE_INFO
-     * XP_SEC_FORTEZZA_BAD_PIN and XP_SEC_FORTEZZA_PERSON_ERROR are 
+     * XP_SEC_FORTEZZA_BAD_PIN and XP_SEC_FORTEZZA_PERSON_ERROR are
      * placeholder for backward compatibility.
      */
+    @Deprecated
```

Comment:
Wouldn't this `@Deprecated` warning also need to be added on the later elements as well? If not, wouldn't you need to mark the `SEC_ERROR` and beyond as not deprecated?

Reply:
I think Java requires a separate deprecation annotation for each constant, so the other constants aren't actually deprecated yet. We probably should do that in a separate PR though.

Reply:
I am making a separate PR to add in these additional annotations. This keeps the situation where we have not deprecated something when we should logically distinct from this PR, where we're simply adding annotations to things we have already deprecated,

Reply:
See #736